### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by DrDelgod** [community/community#55147] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I have never used this service before and someone has been miss using my information without my permission or consent and I am sick and tired of being taken advantage of the last time I checked, it’s called a hate crime.. I’m trying to get to the bottom of these issues.. so your going to help me figure this out or we can take it to the district attorney and see how this ends for y’all

**Comment by queenofcorgis** [comment, marked as answer]
Hi @DrDelgod 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
